store/mockstore/unistore/cophandler/analyze.go

```diff
@@ -350,3 +352,172 @@ func (e *analyzeColumnsExec) NewChunk() *chunk.Chunk {
 func (e *analyzeColumnsExec) Close() error {
 	return nil
 }
+
+func handleAnalyzeMixedReq(dbReader *dbreader.DBReader, rans []kv.KeyRange, analyzeReq *tipb.AnalyzeReq, startTS uint64) (*coprocessor.Response, error) {
```

Comment:
Can we reuse any existing analyze code for mixed analyze logic?

Reply:
I have extracted a function `buildBaseAnalyzeColumnsExec` to reuse some codes.